### ISSUE TITLE
Null Reference error when using Conditional Field

### ISSUE THREAD

**Issue by jfmaniccia** [Deadcows/MyBox#58]
For some reason sometimes when I use the Conditional field I get a null reference error. I have no idea why besides it has something to do with the if statement on line 78 of the conditional field attribute script. If I am reading it right it is checking if I have more than one condition attached to the variable, which I do not. If I am reading it wrong than my bad. But when I comment out line 78 it works perfectly fine.
<img width="1015" alt="Screen Shot 2020-02-06 at 3 32 51 PM" src="https://user-images.githubusercontent.com/33293494/73987722-25b51300-48f6-11ea-907c-a44effc43202.png">
If I have a bunch of variables with a conditional field attached to each the first one with a conditional field will sometimes get the null reference exception.
<img width="892" alt="Screen Shot 2020-02-06 at 3 34 06 PM" src="https://user-images.githubusercontent.com/33293494/73988112-92c8a880-48f6-11ea-9abe-5817cd358743.png">
The variables under base stats work perfectly fine. The variables under specific body stats work fine except the first variable. It throws this null reference error.
<img width="904" alt="Screen Shot 2020-02-06 at 3 35 26 PM" src="https://user-images.githubusercontent.com/33293494/73988118-952b0280-48f6-11ea-8d74-d0fd4f574933.png">

**Comment by Deadcows** [comment]
Well, that's reeehehelly strange issue. I tried to handle every place where nullref could possibly occur in [this commit](https://github.com/Deadcows/MyBox/commit/8beb06d141c76e752dab1600034fa5510075e209#diff-a780fd391b6c42f048d7335f761eb21b) (in _HaveMultipleAttributes_ method).
It would be great if you'll be able to profile this issue on line 78 before the update, I'd like to know what part of the line causes exception. But if it's hard for you to reproduce this feel free to update to the latest commit, should work fine 🤓

Thanks for the report!

**Comment by Deadcows** [comment]
Ok I were able to reproduce nullref exception with fieldInfo somehow is null.
My setup:
        `public bool B;
        [Tag(order = 1), ConditionalField(nameof(B), order = 2)] public string BB;`
It's nothing like your case, but fieldInfo should always have a value so maybe this somehow will help me to figure this out...

**Comment by jfmaniccia** [comment]
After looking through code and the project in general I decided to try and recreate the error in a new project to see if it was something that I could show step by step to do or if there was anything in particular that would trigger the error. I could not recreate it in another project. As of this moment, I have tried everything that I can think of to create this bug again in a different project but it has never shown up. I think there might be something wrong with the project that this error is occurring in so I will look into that.
